python-sdk/example_dags/example_load_file.py

```diff
@@ -253,4 +253,17 @@
     )
     # [END load_file_example_19]
 
+    # [START load_file_example_20]
+    aql.load_file(
+        input_file=File(path="sftp://upload/", conn_id="sftp_conn", filetype=FileType.CSV),
+        output_table=Table(
+            conn_id=SNOWFLAKE_CONN_ID,
+            metadata=Metadata(
+                database=os.environ["SNOWFLAKE_DATABASE"],
```

Comment:
@rajaths010494 Can we add an integration test case where verify table post creation? or extend an existing one.

Reply:
Will change to suggested path

Reply:
will try to export and test this functionality

Reply:
Will add tests for it

Reply:
added test in load_file to read data from sftp docker and validate it